src/coord/coord.rs

```diff
@@ -454,11 +451,19 @@ where
                 }
                 Message::Worker(WorkerFeedbackWithMeta {
                     worker_id: _,
-                    message: WorkerFeedback::CreateSource(source_id, sc, consistency),
+                    message: WorkerFeedback::CreateSource(source_id, _sc),
                 }) => {
-                    ts_tx
-                        .send(TimestampMessage::Add(source_id, sc, consistency))
-                        .expect("Failed to send CREATE Instance notice to timestamper");
+                    if let Some(entry) = self.catalog.try_get_by_id(source_id.sid) {
```

Comment:
I had meant to ask you this @benesch: is this id guaranteed to exist? It seems to be the case, all sources are registered once in the catalog, but just want to double check